src/bundle/DependencyInjection/EzSystemsEzSupportToolsExtension.php

```diff
@@ -76,4 +75,39 @@ private function getPoweredByName(ContainerBuilder $container, ?string $release)
 
         return $name;
     }
+
+    private static function getNameBySubscriptionInfo(string $file): ?string
+    {
+        if (!file_exists($file)) {
+            return null;
+        }
+
+        $subscriptionData = json_decode(file_get_contents($file), true);
+        $name = IbexaSystemInfo::PRODUCT_NAME_VARIANTS['content'];
+        foreach ($subscriptionData['product_additions'] as $product) {
+            // Map older subscription names to new where needed.
+            $identifier = in_array($product['name'], ['enterprise', 'platform']) ? 'experience' : $product['name'];
+            if ($identifier !== 'content') {
+                $name = IbexaSystemInfo::PRODUCT_NAME_VARIANTS[$identifier];
+            }
+
+            // Break out if highest package (commerce) was detected
+            if ($identifier === 'commerce') {
+                break;
+            }
+        }
+
+        return $name;
+    }
+
+    private static function getNameByPackages(): string
+    {
+        if (is_dir($vendor . IbexaSystemInfoCollector::COMMERCE_PACKAGES[0])) {
+            $name = IbexaSystemInfo::PRODUCT_NAME_VARIANTS['commerce'];
+        } elseif (is_dir($vendor . IbexaSystemInfoCollector::ENTERPRISE_PACKAGES[0])) {
+            $name = IbexaSystemInfo::PRODUCT_NAME_VARIANTS['experience'];
+        } else {
```

Comment:
Should add fallback logic for `content` here as well:
- Add constant with 1-2 packages part of content _(+ review other constants to make sure they are still correct)_
- Add logic here for it
- Add logic for it in "shouldHaveSubscription" logic as well: https://github.com/ezsystems/ez-support-tools/pull/77/files#diff-079f6b421ab6d50e09d18ed9dc21c725bdc0ee1832d85ab075ddeac8b16d6766R214